### ISSUE TITLE
Prioritise Jobs for Dual Mining

### ISSUE THREAD

**Issue by SystemtechCA** [cTn-dev/Excavator-GUI#2]
Hi, Is there a way to prioritize jobs for dual mining purposes?
I am running DaggerHashimoto and Sia and the second job(Sia) in my case takes priority over Daggerhash and i want it the other way around. Maybe a dropdown on the jobs for the priority (1-10)?

**Comment by cTn-dev** [comment]
Hi, currently there is no way to prioritize jobs, it should be possible with custom worker parameters, but as of right now, i unfortunately don't have enough knowledge when it comes to tweaking the worker values, if the developers of excavator post some more info about the worker parameters, i could definitely look into it.

**Comment by SystemtechCA** [comment]
Fair enough. If i get some time ill play around with excavator and see if i can find anything:)
